### PR TITLE
Avoid duplicate as3 body validation.

### PR DESCRIPTION
**Pull request by zongzw**
**Description**: 

Performance improvement for as3 body validation.

**Changes Proposed in PR**:

This PR use `AS3Manager.as3Validated` to check the provided template already validated or not.
"Trade space for time".

**Fixes**: 
No issue ID, but performance related.
Related PRs
#2194 
#2292 

## General Checklist
- [x] Smoke testing completed

### CODE REVIEW

**Comment by trinaths** [comment]
Created [CONTCNTR-3241] for internal tracking.

**Comment by sravyap135** [comment]
@zongzw Could you please restrict the scope of this PR to only md5 checksum changes so that I can approve them. 

The timeit changes might help for debugging and we do not want to have them as part of master branch.

File: pkg/appmanager/appManager.go [hunk 1]
```diff
@@ -482,13 +482,7 @@ func (appMgr *Manager) triggerSyncResources(ns string, inf *appInformer) {
 }
 
 func (appMgr *Manager) syncNamespace(nsName string) error {
-	startTime := time.Now()
 	var err error
-	defer func() {
```
Comment (sravyap135) [inline pkg/appmanager/appManager.go:487]: Can you please include this debug log

File: pkg/prometheus/prometheus.go [hunk 2]
```diff
@@ -34,7 +32,6 @@ var CurrentErrors = prometheus.NewGaugeVec(
 // further metrics? todo think about
 // RegisterMetrics registers all Prometheus metrics defined above
 func RegisterMetrics() {
-	log.Info("[CORE] Registered BigIP Metrics")
```
Comment (sravyap135) [inline pkg/prometheus/prometheus.go:37]: let us have this log

**Comment by sravyap135** [comment]
Can you rename the commit..

**Review by sravyap135 — APPROVED** [review]
LGTM

**Comment by zongzw** [comment]
Just get it regulated. Thanks for proceeding and reviewing it.

**Comment by sravyap135** [comment]
Triggering smoke tests on this PR.

**Pull request merged**